test/test_utils.py

```diff
@@ -961,3 +964,14 @@ def test_detect_encoding(self):
             finally:
                 f.close()
                 os.remove(f.name)
+
+    def test_detect_encoding_file(self):
+        test_files = {
+            get_test_data_path('eac-utf16le.log'): 'utf-16-le',
+            # get_test_data_path('eac-utf8.log'): 'utf-8',
```

Comment:
Is there a problem with the utf8 file specifically?

Reply:
oh right, I forgot to mention it. Actually the current detector and a couple of online detectors are considering this log file of type `ascii` and one strange detector considered the type `iso-8859-1`.

Reply:
Ah, yes. I see. Actually they are all correct :) As the data contains only ASCII characters (and no BOM), all of a `ascii`, `iso-8859-1` and `utf-8` result in exactly the same bytes.

I think it's allright to leave this specific case out of the test. We could maybe replace it with a file where UTF-8 encoding actually makes a difference. But we don't need to do this as part of this PR.